### PR TITLE
chore(travis): add config

### LINKED ISSUE #1: Setup CI
I will go with Travis:
* install deps
* run lint and tests

### PR DESCRIPTION
**Pull request by adnasa**
Aight, this is my very first attempt to travis ci... ever! 😓 

#### open question(s)

I simply chose nodejs 8, since there was no engine requirement in `package.json`. Shall we set this to nodejs10 ([ref](https://docs.npmjs.com/files/package.json#engines))

closes #1

### CODE REVIEW

**Comment by montezume** [comment]
I would be interested in moving to node 10... what do you think @emmenko @dferber90 ?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  email: false
+
+cache:
+  yarn: true
+  directories:
+    - node_modules
+
+script:
+  - npm run lint
+  - npm run test
```
[the remark this answers is not in the document]
Comment (adnasa) [reply .travis.yml]: image

you got it

**Comment by adnasa** [comment]
Shall we update the `engine` requirement first?

**Comment by montezume** [comment]
@adnasa or just add one, I don't see any listed.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+sudo: false
+language: node_js
+node_js:
+  - '8'
```
Comment (emmenko) [inline .travis.yml]: Node `10`

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  email: false
+
+cache:
+  yarn: true
+  directories:
+    - node_modules
+
+script:
+  - yarn lint
+  - yarn test
```
Comment (emmenko) [inline .travis.yml]: We're using jest runners, so you can just do `yarn run jest --projects jest.*.config.js`.

PS: the current `yarn lint` command is actually wrong, it should be `jest --projects jest.eslint.config.js jest.stylelint.config.js`

Comment (adnasa) [reply .travis.yml]: looks like travis took care of this?

```
$ yarn lint
yarn run v1.3.2
$ jest --projects jest.*.config.js
```

Comment (emmenko) [reply .travis.yml]: > the current yarn lint command is actually wrong
👆

**Comment by adnasa** [comment]
looks like netlify deployment failed when I set it to `10.8.0`, will try with `10.5.0`

**Comment by emmenko** [comment]
```
2:44:15 PM: Installing NPM modules using Yarn version 1.3.2
2:44:16 PM: yarn install v1.3.2
2:44:16 PM: [1/5] Validating package.json...
2:44:16 PM: error @commercetools-frontend/ui-kit@1.0.0-beta.30: The engine "node" is incompatible with this module. Expected version ">=10.8.0".
2:44:16 PM: error Found incompatible module
2:44:17 PM: info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
2:44:17 PM: Error during Yarn install
2:44:17 PM: Error running command: Build script returned non-zero exit code: 1
```

**Comment by emmenko** [comment]
Keep the `engine` out for now

**Comment by adnasa** [comment]
okay, thanks.
will update the lint command as you asked :)

**Comment by adnasa** [comment]
config added, and `lint` command updated.. anything to add?

File: .travis.yml (hunk 2 above)
Comment (dferber90) [reply .travis.yml]: What about this @adnasa ?

**Comment by emmenko** [comment]
> We're using jest runners, so you can just do `yarn run jest --projects jest.*.config.js`.

Can you change to this for CI?

Also it still runs on node 8. Can you change it to `10`?

**Pull request merged**